lib/tilebag.rb

```diff
@@ -0,0 +1,37 @@
+require 'pry'
+require_relative 'scoring.rb'
+require_relative 'player.rb'
+
+module Scrabble
+  class TileBag
+    attr_reader :default_tiles, :tiles_remaining
+
+    def initialize
+      @default_tiles = {"A" => 9,
+        "B" => 2, "C" => 2,
+        "D" => 4, "E" => 12, "F" => 2,
+        "G" => 3, "H" => 2, "I" => 9,
+        "J" => 1, "K" => 1, "L" => 4,
+        "M" => 2, "N" => 6, "O" => 8,
+        "P" => 2, "Q" => 1, "R" => 6,
+        "S" => 4, "T" => 6, "U" => 4,
+        "V" => 2, "W" => 2, "X" => 1,
+        "Y" => 2, "Z" => 1}
+      end
+      
+      def draw_tiles(num)
+        num.times do
+          @drawn_tiles = []
+          letter = @default_tiles.keys.sample
```

Comment:
I don't think this will work the way you want. Since you're initializing `@drawn_tiles` inside the loop, it'll get reset every time, and your collection of drawn tiles won't ever have more than one letter! Moving line 24 outside the loop should fix this issue.

I also don't like that you're using an instance variable for `@drawn_tiles` - since it's only used in this method, I think a local variable would be more appropriate.